tailwind.config.ts

```diff
@@ -3,10 +3,26 @@ import type { Config } from 'tailwindcss';
 export default {
   content: ['./src/lib/**/*.{ts,tsx}'],
   theme: {
+    aspectRatio: {
```

Comment:
We need to figure out a way to have a single `tailwind.config.ts` file. It would be ideal to modify the vite configuration to also compile this file and have it available in `dist/` as `tailwind.config.js`. That would allow us to import it directly in the Next sites.

Also, see the other comments I left in `public/tailwind.config.ts`.